packages/sources/coinmarketcap/src/endpoint/crypto.ts

```diff
@@ -95,6 +95,7 @@ const presetIds: { [symbol: string]: number } = {
   STETH: 8085,
   '1INCH': 8104,
   WOM: 5328,
+  OHMV2: 9067,
```

Comment:
```suggestion
  OHMV2: 9067,
  OHM: 9067,
```

Reply:
Since we currently use `"coinmarketcap":{"OHMv2":"OHM"}` in the overrides, we should also include `OHM` as a safeguard.